nupic/algorithms/anomaly.py

```diff
@@ -164,6 +168,18 @@ def __str__(self):
     return "Anomaly:\tmode=%s\twindowSize=%r" % (self._mode, windowSize)
 
 
+  def __eq__(self, other):
+    if not isinstance(other, Anomaly): 
+      return False
+    if (other._mode == self._mode and
+        other._binaryThreshold == self._binaryThreshold and
+        other._movingAverage == self._movingAverage and
+        other._likelihood == self._likelihood):
+      return True #equal
+    else:
+      return False
+
+
   def __setstate__(self, state):
```

Comment:
@vitaly-krugl this PR is just a simple "preparation" PR, introducing serialization tests to Anomaly. the `self.computeFn()` or `mode=custom` is not used here. So yes, it should be safe here. Once merged, I'll add check for equality of the computeFn to `__eq__` and we will see.